src/tree/tooltip.js

```diff
@@ -0,0 +1,57 @@
+import { select } from 'd3';
+import constants from './size-constants';
+
+let tooltipOpen = -1;
```

Comment:
global variables :( I think can get weird with multiple orgcharts in sense

Reply:
yeah, that is definitely not working right now :/

Reply:
It can be fine, as long as you never need multiple tooltips at the same time. But you also need to use the same element then, else you'll run into issues not having closed a tooltip.

So I think either scope the tooltip timeouts to the box function again, or move the tooltip element onto the html body. That causes some other issues, so best keep it inside the org chart div.

Reply:
the problem isn't even the opening and closing, when I have two org charts, the tooltip is over the card in the other orgchart than the one I'm hovering over.
but I guess that would be solveable by going back to passing around the reference to the tooltip

Reply:
as well as scoping the timeouts to box I guess

Reply:
you could just build it as a class/object of sorts:
- in index you initialize it with the element and save the tooltip reference to the selectionsAndTransform object:
`selectionsAndTransform.tooltip = tooltip.init(element)` -> that can save timouts on itself etc
- Pass that to box.js
- Profit